core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -94,6 +94,74 @@ object UserService {
       "roles/bigquery.jobUser" -> Set(s"group:${ownerGroupEmail.value}", s"group:${computeUserGroupEmail.value}")
     )
 
+  // TODO - once workspace migration is complete and there are no more v1 workspaces or v1 billing projects, we can remove this https://broadworkbench.atlassian.net/browse/CA-1118
+  def deleteGoogleProjectIfChild(projectName: RawlsBillingProjectName,
```

Comment:
Moved up to companion object to allow sharing from `GoogleBillingProjectLifecycle`.